### PR TITLE
Add og image.

### PR DESCRIPTION
**Pull request by catcatcatcat**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -19,8 +19,15 @@
...
     <!-- Custom styles for this template -->
-    <link href="css/clean-blog.min.css" rel="stylesheet">
-
+    <link href="css/clean-blog.min.css" rel="stylesheet"> 
+    
+    <meta property="og:title" content="BSDTW" />
+    <meta property="og:description" content="Get involved with BSDTW community" />
+    <meta property="og:url" content="https://bsdtw.org" />
+    <meta property="og:type" content="website" />
+    <meta property="og:image" content="https://bsdtw.org/img/BSDTW17-RGB.png" />
```
Comment (lwhsu) [inline index.html]: I think more general filename would be better.

File: index.html [hunk 2]
```diff
@@ -19,8 +19,15 @@
...
+    <meta property="og:image:width" content="2481" />
+    <meta property="og:image:height" content="2448" />
```
Comment (lwhsu) [inline index.html]: Does this really need to be that large? I'm not sure the convention of `og:*` stuffs.

Comment (catcatcatcat) [reply index.html]: I modified the image size, please check if this size is better.

File: index.html (hunk 1 above)
Comment (catcatcatcat) [reply index.html]: Filename modified.

**Pull request merged**